### PR TITLE
Feature/home

### PR DESCRIPTION
**Pull request by emarkees**
### In this project
- React Router was added.
- Three pages were created for easy navigation using the React router

Kindly help review and kindly star. 

Thanks

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
 function App() {
   return (
-    <div>
-      <Quote />
-      <Calculator />
-    </div>
+    <BrowserRouter>
+      <Nav />
+      <Routes>
+        <Route path="/Home" element={<Home />} />
```
Comment (TedLivist) [inline src/App.js]: - [ ] Kindly make the Home component the root route.

  ```
  <Route path="/" element={<Home />} />
  ```

File: src/Component/Nav.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Nav.css';
+
+const Nav = () => (
+  <nav className="nav">
+    <div className="header">Math Magician</div>
+    <ul className="link">
+      <li>
+        <Link to="/Home" className="font">Home</Link>
```
Comment (TedLivist) [inline src/Component/Nav.jsx]: - [ ] Kindly modify the link on the Navbar to be the root path.

  ```
  <Link to="/" className="font">Home</Link>
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @emarkees,

Good job so far!
✔️ Implementation of routes with RRD
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Component/Nav.jsx (hunk 2 above)
Comment (emarkees) [reply src/Component/Nav.jsx]: Thanks for the review

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @emarkees, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**